concent_api/middleman/asynchronous_operations.py

```diff
@@ -186,6 +189,16 @@
     return True
 
 
+async def heartbeat_producer(
+    writer: StreamWriter,
+    request_id: int = HEARTBEAT_REQUEST_ID
```

Comment:
I am wondering if it won't be better to just remove the parameter and use constant directly in code. What do you think?

Reply:
I was wondering about that. Yes, right now there is no need to pass `request_id`. I will remove it.